app/views/provider/admin/quick_starts/show.html.slim

```diff
@@ -0,0 +1,12 @@
+#quick-start-catalog-container
+
+javascript:
+  // Hack of the year, part 2: For QuickStartCatalogPage to work we need to render it as a children
+  // of QuickStartContainer, we can't just render it in another pack here. This moves the node from
+  // within QuickStartContainer to #quick-start-catalog-container
```

Comment:
Was any non-hack solution discussed?

Reply:
No, not really, but if we don't want to use a hack we need to implement the whole thing from scratch. Patternfly-react already takes care of everything BUT it's meant to be used in a React application, which we don't have.
The non-hack option is to use plain React (html/css) and implement ourselves the business logic... it's a no-go.